doc/whats_new/v0.22.rst

```diff
@@ -795,6 +795,14 @@ Changelog
   :pr:`13925` by :user:`Isaac S. Robson <isrobson>` and :pr:`15524` by
   :user:`Xun Tang <xun-tang>`.
 
+:mod:`sklearn.semi_supervised`
+..............................
+
+- |Fix| :class:`semi_supervised.LabelPropagation` and
```

Comment:
Since this is a bug fix, this should be moved up to **Version 0.22.1** in this file.